runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/ReadTranslation.java

```diff
@@ -136,13 +132,9 @@ private static SdkFunctionSpec toProto(UnboundedSource<?, ?> source) {
   public static UnboundedSource<?, ?> unboundedSourceFromProto(ReadPayload payload)
       throws InvalidProtocolBufferException {
     checkArgument(payload.getIsBounded().equals(IsBounded.Enum.UNBOUNDED));
-    return (UnboundedSource<?, ?>) SerializableUtils.deserializeFromByteArray(
-        payload
-            .getSource()
-            .getSpec()
-            .getPayload()
-            .toByteArray(),
-        "BoundedSource");
+    return (UnboundedSource<?, ?>)
+        SerializableUtils.deserializeFromByteArray(
+            payload.getSource().getSpec().getPayload().toByteArray(), "BoundedSource");
```

Comment:
<!--new_thread; commit:042367c660b607e55c9faafa4d3d8a8685547151; resolved:0-->
While you're here, fix this